addons/website/static/src/scss/primary_variables.scss

```diff
@@ -2094,8 +2094,9 @@ $o-website-values-palettes: (
         // By default, the $o-base-website-values-palette for fonts are "null"
         // but will actually automatically be filled with the first font themes
         // define in $o-theme-font-configs (for the base one). For the default
-        // website, we want to specifically use the system fonts.
-        'font': 'SYSTEM_FONTS',
+        // website, we use a combination of Figtree for headings and Inter.
+        'headings-font': 'Figtree',
```

Comment:
It's strange to sometimes have a small white line on headings with `Figtree`  :thinking: (only for some specific chars: n, h, u…):
![master rev-website-default-font--chgo](https://github.com/user-attachments/assets/25def067-37c6-4322-96f0-3e78f47bbf07)